docker/chemml/src/run.py

```diff
@@ -0,0 +1,49 @@
+import json
+from chemml.models.keras.trained import OrganicLorentzLorenz
+
+from openbabel import OBMol, OBConversion
+
+def ob_convert_str(str_data, in_format, out_format):
+    mol = OBMol()
+    conv = OBConversion()
+    conv.SetInFormat(in_format)
+    conv.SetOutFormat(out_format)
+    conv.ReadString(mol, str_data)
+
+    return (conv.WriteString(mol), conv.GetOutFormat().GetMIMEType())
+
+def run_calculation(geometry_file, output_file):
+    # In the future we can probably accept SMILES directly from the molecule
+    # model. For now we need somewhere to put the output, so the CJSON makes
+    # more sense.
+    with open(geometry_file) as f:
+        cjson = json.load(f)
+
+    smiles = cjson_to_smiles(cjson)
+
+    cls = OrganicLorentzLorenz()
+    cls.load(summary=True)
+    pred = cls.predict(smiles)
+
+    calculated_properties = {
```

Comment:
I believe @cryos wanted these calculated quantities in the form `"name": number` and to keep the information about labels and units into a separate table.
Also, they should now go under the `properties` key rather than `calculatedProperties`

Reply:
Ok, can you push a commit to fix this up?

Reply:
Sure!